### PR TITLE
Create fishery-bulletin.csl

### PR DESCRIPTION
**Pull request by POBrien333**
via https://forums.zotero.org/discussion/68014/style-request-fishery-bulletin

### CODE REVIEW

File: fishery-bulletin.csl [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+          <else>
+            <text macro="title"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="symbol" delimiter=", " initialize-with=". "/>
```
Comment (adam3smith) [inline fishery-bulletin.csl]: should be and="text"?

File: fishery-bulletin.csl [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+        </names>
+      </if>
+    </choose>
+  </macro>
+  <macro name="secondary-contributors">
+    <choose>
+      <if type="chapter paper-conference" match="none">
+        <names variable="translator editor" delimiter=", " prefix=" (" suffix=")">
+          <name and="symbol" initialize-with=". " delimiter=", "/>
+          <label form="short" prefix=", " text-case="title"/>
```
Comment (adam3smith) [inline fishery-bulletin.csl]: you're sure the labels for editors are different for chapters (above: lowercase, stripped periods) and books (uppercase, include periods)? Absent guidance, I'd unify this.

File: fishery-bulletin.csl [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+      <group delimiter=". ">
+        <text macro="issued"/>
+        <text macro="title"/>
+        <text macro="container"/>
+        <group delimiter=", " suffix=".">
+          <text macro="event"/>
+          <text macro="publisher"/>
+        </group>
+      </group>
+      <text macro="locators" suffix="."/>
```
Comment (adam3smith) [inline fishery-bulletin.csl]: Styleguide suggests DOI for all published work: "doi  code should  be  included  at  the  end  of  citations  of  published  literature. " https://web.archive.org/web/20170504204954/http://fishbull.noaa.gov/guidelines.pdf

Comment (adam3smith) [reply fishery-bulletin.csl]: The checklist says the same. Use https://doi.org/ as prefix. If you look at the published article, they actually use "Article" with a doi-based link at the end of all applicable citations, so this does get used.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**